src/components/search/Search.jsx

```diff
@@ -92,8 +92,7 @@ const Search = () => {
   const shouldDisplayBalanceAlert = hasNoEnterpriseOffersBalance || hasLowEnterpriseOffersBalance;
 
   const { content_type: contentType } = refinements;
-  const hasRefinements = Object.keys(refinements).filter(refinement => refinement !== 'showAll').length > 0;
-
+  const hasRefinements = Object.keys(refinements).filter(refinement => refinement !== 'showAll').length > 0 && contentType?.length > 0;
```

Comment:
`contentType?.length` The question mark is relevant to stay because by default, contentType is undefined